package.json

```diff
@@ -12,7 +12,6 @@
     "index.d.ts"
   ],
   "scripts": {
-    "prepublish": "npm test",
```

Comment:
I've removed this task, as newer versions of NPM skip this hook, you can see this by running `npm publish --dry-run`. Additionally, the tests already run when PRs are merged, so this should not be an issue.